openedx/features/colaraz_features/middlewares.py

```diff
@@ -59,11 +59,11 @@ def _redirect_to_login(self, request):
         if third_party_auth.is_enabled() and backend_name:
             provider = [enabled for enabled in third_party_auth.provider.Registry.enabled()
                         if enabled.backend_name == backend_name]
-
-            if not provider and configuration_helpers.get_value('AUTH_PROVIDER_FALLBACK_URL'):
-                fallback_url = configuration_helpers.get_value('AUTH_PROVIDER_FALLBACK_URL')
+            fallback_url = configuration_helpers.get_value('AUTH_PROVIDER_FALLBACK_URL') \
+                or configuration_helpers.get_value('LMS_BASE')
+            if not provider and fallback_url:
                 next_url = urlencode({'next': self._get_current_url(request)})
-                redirect_url = '{}?{}'.format(fallback_url, next_url)
+                redirect_url = '//{}?{}'.format(fallback_url, next_url)
```

Comment:
`//` forward slashes at the start of URL?

Reply:
If we use '//' in start of URL, Django will append HTTP or https itself, and redirection will use whole URL. Otherwise if URL is '/abc' it will be appended with current domain which will not redirect to other domain.

Reply:
Okay 👍